setup.py

```diff
@@ -16,12 +16,13 @@
     py_modules=[],
     install_requires=[
         "autobahn",
-        "bitcoinlib==0.4.5",
+        "bitcoinlib>=0.4.14",
```

Comment:
Maybe we should make the require bitcoinlib version _equal_ to `0.4.14` and not equal or bigger.

Reply:
Yeah, the question is whether we play it safe and keep the user stuck to the eventually old version, or do we let bitcoinlib get better and only once an incompatibility appears, we jump in to fix it.

Reply:
```suggestion
        "bitcoinlib==0.4.14",
```